### PR TITLE
Setup MongoDB

### PR DESCRIPTION
**Pull request by pmadridg**
Setup MongoDB

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -11,14 +13,17 @@ app.use(logger('dev'));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
+app.use(bodyParser.json());
+mongoose.connect('mongodb://robinson:igashinoeden2019@ds253567.mlab.com:53567/fugudb', { useNewUrlParser: true });
+var db = mongoose.connection;
+
```
Comment (robinsondotnet) [inline app.js]: remove unnecessary empty lines

File: models/recipeModel.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+var mongoose = require('mongoose');
+// Setup schema
```
Comment (robinsondotnet) [inline models/recipeModel.js]: Remove this comment

File: models/recipeModel.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+const recipeSchema = mongoose.Schema({
+    id: {
+        type: String,
+        require: true
+    },
+    name:{
+        type: String,
+        require: true
+    },
+
```
Comment (robinsondotnet) [inline models/recipeModel.js]: Remove this empty line

Comment (daynine23) [reply models/recipeModel.js]: HAHAHAHAHA

**Pull request merged**